lib/core/repository/tip_repository.dart

```diff
@@ -88,9 +88,8 @@ class TipRepository {
 
   Future<void> toggleFavouriteTip(Tip tip) async {
     final tipToUpdate = _tipMapper.fromOutput(tip);
-    tipToUpdate.favouriteDate != null
-        ? tipToUpdate.favouriteDate = null
-        : tipToUpdate.favouriteDate = DateTime.now();
+    tipToUpdate.favouriteDate =
+        tipToUpdate.favouriteDate != null ? null : DateTime.now();
```

Comment:
As a rule, try to use the positive way
```suggestion
    tipToUpdate.favouriteDate =
        tipToUpdate.favouriteDate == null ? DateTime.now() : null;
```